ethereum/balancer_v1/view_pools_fees.sql

```diff
@@ -0,0 +1,8 @@
+CREATE OR REPLACE VIEW balancer_v1.view_pools_fees AS
```

Comment:
This `VIEW` times out wen I try to run it.

Reply:
Are you sure? I can run the statements below - and it returns <5k rows, shouldn't be an expensive operation.

Reply:
Sorry, my bad @mendesfabio. It is not this view, but the `view_transfers_bpt.sql` that is very slow / times out when I try to run in the UI: https://dune.xyz/queries/196397

Reply:
no problem! in fact this view has a lot of rows, so maybe trying to select all its results through the UI won't work. I just created it using dune_user_generated and it went well: https://dune.xyz/queries/197448. what do you think?